deployment/modules/cloudflare/account/r2.tf

```
@@ -0,0 +1,5 @@
+resource "cloudflare_r2_bucket" "mich-backups" {
+  account_id = var.cloudflare_account_id
+  name       = "mich-backups"
+  location   = "weur"
+}
```

```suggestion
resource "cloudflare_r2_bucket" "tf-state-database-backups" {
  account_id = var.cloudflare_account_id
  name       = "tf-state-database-backups"
  location   = "weur"
}
```

No other backups for now, but I prefer to do a bucket per backup target.